packages/replay-next/components/elements/ElementsListData.ts

```diff
@@ -33,9 +35,12 @@ export class ElementsListData extends GenericListData<Item> {
     this._pauseId = pauseId;
     this._replayClient = replayClient;
 
-    this._rootObjectIdWaiter = {} as any;
+    this._rootObjectIdWaiter = {
+      resolved: false,
+    } as any;
     this._rootObjectIdWaiter.promise = new Promise(resolve => {
       this._rootObjectIdWaiter.resolve = resolve;
+      this._rootObjectIdWaiter.resolved = true;
```

Comment:
I guess the intention was to wait for the promise to be resolved before setting `resolved` to `true` but this is setting it immediately when the promise is created.

Reply:
Geeze, you're right. Thanks for catching.